### ISSUE TITLE
Add preservica_representation_name property to Parent Object

### ISSUE THREAD

**Issue by mikeapp** [yalelibrary/YUL-DC#1891]
**Story**

A new requirement came up that will require an additional field on the parent. The new field will indicate the name of the Representation that we want to use. For example:

```
<Representations>
        <Representation type="Preservation" name="Preservation-1"
            >https://preservica-dev-v6.library.yale.edu/api/entity/information-objects/1e42a2bb-8953-41b6-bcc3-1a19c86a5e3d/representations/Preservation/1</Representation>
    </Representations>
```

This list could have multiple entries with `name`s like Preservation-2 or Access-1. We will need to know which one to choose when ingesting or resyncing the children of this parent.

**Acceptance**
- [x] The parent object model has a string property `preservica_representation_name`
- [x] The value is displayed on the parent object show page, after "Preservica URI".  It should have the label "Preservica Representation Name"
- [x] The field is editable on the parent object edit page.   Layout as shown:

![Screen Shot 2022-02-08 at 1.31.06 PM.png](https://images.zenhubusercontent.com/5e5d472f410278efd81466ab/2964f295-c8df-4628-9ebd-9196405822fd)

**Comment by jpengst** [comment]
PR: https://github.com/yalelibrary/yul-dc-management/pull/922

**Comment by FCRodriguez7** [comment]
Released management v2.58.23

**Comment by motropuk** [comment]
Looks good on UAT
